bors.toml

```diff
@@ -7,7 +7,8 @@ status = [
     "test (x86_64-unknown-linux-gnu, ubuntu-latest)",
     "test (x86_64-apple-darwin, macos-latest)",
     "docker_tests (happy_path)",
-    "docker_tests (happy_path_restart_bob_before_comm)",
+    "docker_tests (given_bob_restarts_after_xmr_is_locked_resume_swap)",
+    "docker_tests (given_bob_restarts_before_xmr_is_locked_resume_swap)",
```

Comment:
So far we used file names to identify the running test task - do test names actually work?

Reply:
Additionally: I think you forgot to add these tests to CI - or was that on purpose only changed in bors.toml?

Reply:
Yep, I misunderstood how this was all set up :D